improver_tests/blending/weights/test_ChooseWeightsLinear.py

```diff
@@ -328,43 +386,59 @@ def test_3d_array(self):
         self.assertArrayAlmostEqual(weights, expected_weights)
 
 
-class Test__create_new_weights_cube(IrisTest):
-    """Test the _create_new_weights_cube function. """
+"""Test the _create_new_weights_cube function. """
```

Comment:
This is no longer a doc-string and has become an active line with no purpose. Would this be better?
```suggestion
# Test the _create_new_weights_cube function
```

Reply:
Sure.